continuous_integration/test_script.sh

```diff
@@ -8,6 +8,11 @@
 
 set -e
 
+# Get into a temp directory to see if we generate artefacts
```

Comment:
Isn't this rather "to see if tests run from the installed scikit-learn"? develop leaves it inplace and adds it to the path, right?